community/bdwgc/sources

```diff
@@ -0,0 +1 @@
+https://github.com/ivmai/bdwgc/releases/download/v8.0.4/gc-8.0.4.tar.gz
```

Comment:
https://hboehm.info/gc/gc_source/gc-8.0.4.tar.gz

Can drop `autogen.sh` by using this source.